### DISCUSSION TITLE
Auto-slide paused by default?

### DISCUSSION THREAD

**Discussion by jaybonthius** [hakimel/reveal.js#3125] [Q&A] [answered]
Is there a way to set [`autoSlide`](https://revealjs.com/auto-slide/) to be paused by default? I've searched through the documentation and haven't found anything.

**Comment by hakimel** [comment, marked as answer]
```js
Reveal.initialize().then( () => {
  Reveal.configure({ autoSlide: 10000 });
  Reveal.toggleAutoSlide( false );
} );
```

That will wait until reveal.js ready before enabling and immediately pausing `autoSlide`. Hope it helps!
